addons/mail/static/src/components/attachment/attachment_tests.js

```diff
@@ -595,6 +595,34 @@ QUnit.test('close attachment viewer', async function (assert) {
     );
 });
 
+QUnit.test('close attachment viewer before image load', async function (assert) {
+    assert.expect(1);
+
+    await this.start({ hasDialog: true });
+    const attachment = this.env.models['mail.attachment'].create({
+        filename: "test.png",
+        id: 750,
+        mimetype: 'image/png',
+        name: "test.png",
+    });
+    await this.createAttachmentComponent(attachment, {
+        detailsMode: 'hover',
+        isDownloadable: false,
+        isEditable: false,
+    });
+    await afterNextRender(() => document.querySelector('.o_Attachment_image').click());
+    const image = document.querySelector('.o_AttachmentViewer_viewImage');
+    await afterNextRender(() =>
+        document.querySelector('.o_AttachmentViewer_headerItemButtonClose').click()
+    );
+    image.dispatchEvent(new Event('load', { bubbles: true }));
```

Comment:
Don't use variable `image`, especially when obtained before `afterNextRender()` and used `afterNextRender()`.

Reply:
It is on purpose, I need a reference to image before it is removed from the DOM, I add a comment for that.

Reply:
Ok, this piece of information could be provided in heading comment of this technical test too.